src/components/select/Select.js

```diff
@@ -61,10 +62,14 @@ const Select = ({
                             <span className="select-overlay" />
                         </span>
                         {infoTooltip && (
-                            <Tooltip position="middle-right" text={infoTooltip}>
-                                <span className="tooltip-icon-container">
+                            <Tooltip isShown={infoTooltipIsOpen} position="middle-right" text={infoTooltip}>
+                                <PlainButton
```

Comment:
Are the styles for the button the same before & after? Could you add a screenshot to show that?